packages/api/src/types/storage.ts

```diff
@@ -9,6 +9,7 @@ import { AnyFunction, Callback, Codec, CodecArg } from '@polkadot/types/types';
 import StorageKey, { StorageEntry } from '@polkadot/types/primitive/StorageKey';
 
 import { ApiTypes, MethodResult, ObsInnerType, PromiseOrObs, UnsubscribePromise } from './base';
+import { PaginationOptions } from '.';
```

Comment:
Place this in base rather - so `index.ts` doesn't import storage and storage import index.ts. Alternatively, just place it directly in here since it only affects the storage keys.